### ISSUE TITLE
TypeError: Cannot read property 'error_user_msg' of undefined

### ISSUE THREAD

**Issue by supasate** [facebook/facebook-nodejs-ads-sdk#20]
SDK crashes when network error occurs with the following error.

``` bash
TypeError: Cannot read property 'error_user_msg' of undefined
    at new FacebookRequestError (/Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/facebook-nodejs-ads-sdk/dist/cjs.js:340:25)
    at /Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/facebook-nodejs-ads-sdk/dist/cjs.js:466:15
    at tryCatcher (/Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/bluebird/js/release/promise.js:689:18)
    at Async._drainQueue (/Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/Users/supasate/try/try-nodejs-ads-sdk-original/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
```

Reproduction steps:
============
1. Turn off network
2. Make a call via SDK e.g. create or read a campaign

**Comment by eranbetzalel** [comment]
The SDK could not handle native exceptions.

I have a fix for that.

https://github.com/exposebox/facebook-nodejs-ads-sdk/commit/86e1a3c584ede6873274498b55a23886ba4c77d7#diff-ad91291e33bac51ef066bac01e9dfa45L27

**Comment by supasate** [comment]
Fixed in [v2.11.3](https://github.com/facebook/facebook-nodejs-ads-sdk/releases/tag/v2.11.3).
